lib/node_modules/@stdlib/array/base/cunone-by/lib/main.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+// MODULES //
+
+var isAccessorArray = require( '@stdlib/array/base/assert/is-accessor-array' );
+var accessorGetter = require( '@stdlib/array/base/accessor-getter' );
+
+// MAIN //
+
+/**
+ * Cumulatively tests whether no array elements pass a test implemented by a predicate function.
+ *
+ * @param {Array|TypedArray} x - input array
+ * @param {Function} predicate - predicate function
+ * @returns {Array} output array
+ *
+ * @example
+ * function isPositive( value ) {
+ *     return ( value > 0 );
+ * }
+ *
+ * var x = [ 0, 0, 0, 1, 0 ];
+ *
+ * var y = cunoneBy( x, isPositive );
+ * // returns [ true, true, true, false, false ]
+ */
+function cunoneBy( x, predicate ) {
+    var getter;
+    var out;
+    var i;
+
+    if ( isAccessorArray( x ) ) {
+        getter = accessorGetter( x );
+    }
+
+    out = new Array( x.length );
```

Comment:
Do not preallocate arrays like this. You can cause the array to be allocated in object mode. Instead use [] and push.